packages/manager/modules/navbar/src/notifications-menu/template.html

```diff
@@ -8,15 +8,165 @@
 <oui-navbar-dropdown
     data-ng-if="!$ctrl.isLoading"
     name="notifications"
-    data-text="{{:: $ctrl.menuTitle }}"
     data-icon-class="oui-icon oui-icon-bell"
     data-icon-animated=":: $ctrl.iconIsAnimated"
     data-on-click="$ctrl.acknowledgeAll()"
 >
-    <oui-navbar-notification
-        data-links="::$ctrl.sublinks"
-        data-align="end"
-        data-fixed
+    <div
```

Comment:
Can't you create components instead ? This would make everything easier to read and maybe reuse :)

Reply:
This code can not be reused. It's specific to the navbar and used only in this place. It also has a dependency on its parent controller (navbarCtrl: '^^ouiNavbar',) to toggle navbar menu.

Reply:
Could we use something like a callback or an event instead of requiring the parent element ? Even though this component is specific we might want to be able to reuse it somewhere else

Reply:
removed reference to parent component controller navbarCtrl

Reply:
@FredericEspiau This concern has been extracted https://github.com/ovh/manager/pull/2427